### DISCUSSION TITLE
Topic Discovery

### DISCUSSION THREAD

**Discussion by sbodese** [dotnet/MQTTnet#1736] [Q&A]
Hi, 
thanks for this great software!

One question please:
 It is possible to do a topic discovering with this client and use this result to filter out the desired topics, to instantiate further separate mqtt clients for each topic or exists a another approach to do this?

Thanks & Greetings Steffen

**Comment by chkr1011** [comment]
You may try out mqttMultimeter. It has a topic explorer which tracks every published topic so you can identify obsolete ones.
 https://github.com/chkr1011/mqttMultimeter
